lib/rpc/endpoints/spec.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Soroban.RPC.Endpoint.Spec do
+  @moduledoc """
+  Defines RPC response type constructions.
+  """
+  @type response :: tuple()
+  @type transaction :: String.t()
+
+  @callback request(transaction :: transaction()) :: response()
```

Comment:
Comments to consider:
1. `response` type can be more specific than just a `tuple()`. It can return sucessful or error tuples.
2. The `request/1` callback will accept several types, string or keyword list (for https://soroban.stellar.org/api/methods/getEvents in the future...)

Reply:
In this method https://soroban.stellar.org/api/methods/getHealth no parameters are required, so `nil` is a valid value for the `request/1` function, too